modules/ROOT/pages/configure-client-management-azure.adoc

```diff
@@ -0,0 +1,70 @@
++
+. Enter a name and description for your client provider.
+. Enter the following values from your identity provider’s configuration:
+.. *Issuer*: URL that the OpenID provider asserts is its trusted issuer.
+.. *Tenant ID for Azure AD*: The tenant ID from Azure Active Directory. For more information on obtaining your tenant ID, see https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant[How to find your Azure Active Directory tenant ID].
+.. *Authorization Header*: The header to provide credentials to authenticate with a server. This header is required if the provider restricts registration requests to authorized clients.
+. Expand the *Advanced Settings* section. The following selections are optional:
+.. *Disable server certificate validation*: Disables server certificate validation if your Azure client management instance presents a self-signed certificate, or one signed by an internal certificate authority.
+.. *Enable client deletion in Anypoint Platform*: Enables deletion of clients created with this integration.
+.. *Enable client deletion and updates in IdP*: To use this option, you must also select the *Enable client deletion in Anypoint Platform* option. This option enables you to update and delete external clients in the configured IdP through an outbound call made by Anypoint Platform to `{clientRegistrationUrl}/{clientID}`. +
+  * The `client_id` request parameter that is passed is the same as the `client_id` passed in the `PUT` request payload.
+  * The `token_endpoint_auth_method` passed in the payload is always `client_secret_basic`.
+  * Only the `Client Registration URL` and `Authorization Header` come from what is configured in Azure *Client Provider*.
++
+The following example shows the payload for a PUT (update) request:
++
+----
+PUT /oauth2/connect/register/{{client_id}}
+Accept: application/json
+Host: identity.example.com
+Authorization: Bearer access-token
+ {
+     "client_id": "client_id",
+     "client_secret": "some-secret",
+     "redirect_uris": [
+          "https://example.com/"
+      ],
+     "grant_types": [ "authorization_code" ],
+     "token_endpoint_auth_method": "client_secret_basic",
+     "response_types": [ "code" ],
+     "client_name": "test-client-name"
+}
+----
++
+[NOTE]
+`client_name` in the request maps to `name` on the inbound side. +
+The authorization method depends on the grant type. For example, if the grant type is `implicit`, the `id_token` and `token` are returned.
++
+The following example shows the DELETE request header:
++
+----
+DELETE /oauth2/connect/register/{{client_id}}
+Host: identity.example.com
+Authorization: Bearer access-token
+----
++
+[IMPORTANT]
+Neither the `PUT` nor the `DELETE` are expected to work if the call is used directly against an IDP. It's intended for interception by pointing to an application under your control, so you can implement the correct deletion and update APIs for their particular provider.
```

Comment:
We do not need to add the PUT and DELETE examples here. since this is only for Azure and we can put Azure doc here in order to keep updated. 
Update: https://docs.microsoft.com/en-us/graph/api/application-update?view=graph-rest-1.0&tabs=http
Delete: https://docs.microsoft.com/en-us/graph/api/application-delete?view=graph-rest-1.0&tabs=http